pandas/tools/pivot.py

```diff
@@ -75,6 +75,11 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     Returns
     -------
     table : DataFrame
+
+    See also
+    --------
+    DataFrame.pivot : pivot without aggregation that can handle
+        non-numeric data
```

Comment:
Ah, yes, that's good. But not yet the `pd.pivot` one, and that is apparently located here: https://github.com/pandas-dev/pandas/blob/master/pandas/core/reshape.py#L339